modules/fedimint-mint-server/src/lib.rs

```diff
@@ -62,9 +62,7 @@ pub struct MintGenParams {
     pub mint_amounts: Vec<Amount>,
 }
 
-impl ModuleGenParams for MintGenParams {
-    const MODULE_NAME: &'static str = "mint";
-}
+impl ModuleGenParams for MintGenParams {}
```

Comment:
this impl that doesn't do anything is a little weird

Reply:
It does do something. It makes `MintGenParams` implement `ModuleGenParams` trait and thus ... special. :D